IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcAlignmentSegment/Documentation.md

```diff
@@ -0,0 +1,7 @@
+An IfcAlignmentSegment is a segment of an IfcAlignment where either the vertical direction or horizontal direction or cant (in the case of trackdesign) obey a unique mathematical description as a function of chainage where chainage is the longitudinal distance along the horizontal projection of the whole alignment.
```

Comment:
Introducing chainage, where at other points we worked so hard to remove it is counter intuitive.
```suggestion
An IfcAlignmentSegment is a segment of an IfcAlignment where either the vertical or horizontal direction or cant (in the case of trackdesign) obey a unique mathematical description as a function of the horizontal projection segment length of the alignment.
```